packages/core/src/components/Table/Table.spec.tsx

```diff
@@ -38,4 +41,45 @@ describe('Table', () => {
     expect(table).toBeInTheDocument();
     expect(cells.length).toEqual(9);
   });
+
+  it('Render Sorting table', () => {
```

Comment:
Should we move test cases related to the sorting into a nested `describe` block? e.g. 
```
describe('Table', () => {
     describe('SortingTable', () => {
       // ...
     })
  })
```

Reply:
Yes, we can move